docs/SlavaUkraini/examples.md

```diff
@@ -33,6 +33,14 @@ description="In this pretty tutorial, you'll learn how to easily define your own
 %}
 
 
+{% include tutorial_link.html
+page="accessibility/"
+thumbnail="accessibility/thumbnail.png"
+title="Accessible maps"
+description="A guide to basic Leaflet accessibility - make your maps usable to persons of a wide range of abilities."
+%}
```

Comment:
The link to this tutorial/guideline is strategically placed after some of the basic tutorials.